content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Newrelic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `Newrelic` trigger that scales based on a Newrelic metric.
+
+```yaml
+triggers:
+  - type: newrelic
+    metadata:
+      # Required: nrAccount - Subaccount to run the query on
+      nrAccount: 1234567
```

Comment:
What does "nr" mean? We tend to avoid abbreviations.

Reply:
Yep see https://github.com/marcelobartsch/keda/pull/1/files where its a abbreviation of the newrelic name to nr. I've sent a patch to remove the needs as the metadata is scaler specific so using more generic names such as Account is appropriate. Once that's merged between the main coder and myself we'll get this pr updated to reflect the changes for the nr removal.

Reply:
documentation is now updated to reflect the most recent changes and merge of the changes in the scaler configuration @tomkerkhove @JorTurFer. Should align better. Appears we have some odd merge behavior in the scaler pr that is pulling some odd added changes, but when looking at the actual changes wanted this documentation should line up now.

Reply:
We plan to release the v2.5 in 2-3 days. When we release v2.5, we will create the new folders for the next version.
Once it's done, you could merge main branch into your fork and place the files there :) 
We could help you also if you want 😄